Plugin/Catalog/Ui/DataProvider/Product/Form/Modifier/EavPlugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
```

Comment:
This plugin should probably be removed, looks like a copy-paste of another module ;)

Reply:
Or maybe just change the wording to indicate this attribute is linked to a product label. There is no danger of any data loss with this module so we should not display a message like this.

Reply:
Done!